src/main/java/io/jsonwebtoken/BadAudienceJwtException.java

```diff
@@ -0,0 +1,11 @@
+package io.jsonwebtoken;
+
+public class BadAudienceJwtException extends ClaimJwtException {
```

Comment:
Hah, you and me both - I agonize over naming because I know how much it contributes to readability and intuitive APIs...